lib/sous-chef.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * This class is designed to parse the "What's cooking" mails sent to
+ * the Git mailing list about twice a week by the Git maintainer.
+ */
+
+export class SousChef {
+    public readonly mbox: string;
+    public readonly messageID: string | undefined;
+    public readonly branches: {
+        [branchName: string]: {
+            merged: string | undefined;
+            sectionName: string;
+            text: string;
+        }
+    };
+
+    public constructor(mbox: string) {
+        this.mbox = mbox;
+        this.branches = {};
+
+        const sections = mbox.split(/^-{10,}\n\[([^\]]+)\]\n/mg)
```

Comment:
Cool use of regex on the split.  Missing `;`?

Reply:
Yep!